packages/jwk-did-provider/package.json

```diff
@@ -9,6 +9,7 @@
   },
   "dependencies": {
     "@ethersproject/random": "^5.6.1",
+    "@sphereon/ssi-types": "^0.8.0",
```

Comment:
Is there also a reference value in tsConfig?

Reply:
added